threediresultstyler/README.rst

```diff
@@ -14,3 +14,55 @@ The main features are:
 - Add loaded 3Di model as a QGIS SpatiaLite connection
 - Save and load Layer tree presets containing expanded/collapsed groups and layers, visibility and filters
 - Highlight Courant numbers of flowlines above a given threshold for analysis of time step reduction
+
+Before using this tool make sure you have loaded a 3Di model using the 3Di plugin.
+
+.. image:: images/toolbar.PNG
```

Comment:
Om problemen à la 3di documentatie te gebruiken: dit soort bestandsnamen beter allemaal lowercase maken (zowel op disk als in de documentatie).